### PR TITLE
fix: rfox unchained-client parser local ci

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

Makes CI happy again no matter the env

## Pull Request Type

- [x] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

> What protocols, transaction types or contract interactions might be affected by this PR?

### Engineering

- `yarn && yarn env app && yarn test` is happy
- `yarn && yarn env dev && yarn test` is happy
- This PR's CI is happy

### Operations

None

### CODE REVIEW

File: packages/unchained-client/src/evm/arbitrum/parser/__tests__/arbitrum.test.ts [hunk 1]
```diff
@@ -30,7 +30,7 @@ import zrxTradeUsdcToWbtc from './mockData/zrxTradeUsdcToWbtc'
 
 vi.hoisted(() => {
   vi.stubEnv('REACT_APP_FEATURE_NFT_METADATA', 'true')
-  vi.stubEnv('RFOX_PROXY_CONTRACT_ADDRESS', '0xaC2a4fD70BCD8Bab0662960455c363735f0e2b56')
+  vi.stubEnv('REACT_APP_RFOX_PROXY_CONTRACT_ADDRESS', '0xac2a4fd70bcd8bab0662960455c363735f0e2b56')
```
Comment (gomesalexandre) [inline packages/unchained-client/src/evm/arbitrum/parser/__tests__/arbitrum.test.ts:33]: i.e correct env var, and lowercased vs. checksummed

File: packages/unchained-client/src/evm/arbitrum/parser/__tests__/arbitrum.test.ts [hunk 2]
```diff
@@ -918,7 +919,7 @@ describe('parseTx', () => {
             assetId: 'eip155:42161/erc20:0xf929de51d91c77e42f5090069e0ad7a09e513c73',
             components: [{ value: '1000000000000000000' }],
             from: address,
-            to: process.env.RFOX_PROXY_CONTRACT_ADDRESS!,
+            to: '0xaC2a4fD70BCD8Bab0662960455c363735f0e2b56',
```
Comment (gomesalexandre) [inline packages/unchained-client/src/evm/arbitrum/parser/__tests__/arbitrum.test.ts:922]: Previously undefined which was accidentally happy

File: packages/unchained-client/src/evm/arbitrum/parser/__tests__/arbitrum.test.ts [hunk 3]
```diff
@@ -989,7 +990,7 @@ describe('parseTx', () => {
           {
             assetId: 'eip155:42161/erc20:0xf929de51d91c77e42f5090069e0ad7a09e513c73',
             components: [{ value: '4364832128906250000' }],
-            from: process.env.RFOX_PROXY_CONTRACT_ADDRESS!,
+            from: '0xaC2a4fD70BCD8Bab0662960455c363735f0e2b56',
```
Comment (gomesalexandre) [inline packages/unchained-client/src/evm/arbitrum/parser/__tests__/arbitrum.test.ts:993]: Previously undefined which was accidentally happy

File: packages/unchained-client/src/evm/arbitrum/parser/index.ts [hunk 4]
```diff
@@ -10,7 +10,6 @@ import * as rfox from '../../parser/rfox'
 import * as zrx from '../../parser/zrx'
 
 export const ZRX_ARBITRUM_PROXY_CONTRACT = '0xDef1C0ded9bec7F1a1670819833240f027b25EfF'
-export const RFOX_PROXY_CONTRACT_ADDRESS = '0xaC2a4fD70BCD8Bab0662960455c363735f0e2b56'
```
Comment (gomesalexandre) [inline packages/unchained-client/src/evm/arbitrum/parser/index.ts:13]: Unused

Comment (kaladinlight) [reply packages/unchained-client/src/evm/arbitrum/parser/index.ts:13]: Feels like we could think about a contracts package as well that is a single source of truth for all things contracts. We have addresses spread all over the place, and also just noticing, we haven't update the parser with newest fox staking contracts and therefore aren't parsing new txs. Would help with scenarios such as that too.

Comment (gomesalexandre) [reply packages/unchained-client/src/evm/arbitrum/parser/index.ts:13]: Absolutely agree about a contracts packages! We already have a `contracts` folder in web so should be an easy lift to just start by moving stuff out of it

Comment (kaladinlight) [reply packages/unchained-client/src/evm/arbitrum/parser/index.ts:13]: https://github.com/shapeshift/web/issues/7290

**Review by kaladinlight — APPROVED** [review]

**Pull request merged**